apps/payload/src/collections/skills/currently-learning.ts

```diff
@@ -0,0 +1,18 @@
+import type { CollectionConfig } from 'payload/types'
+
+const CurrentlyLearning: CollectionConfig = {
+  slug: 'currently-learning',
+  fields: [
+    {
+      name: 'currentSkill',
+      type: 'text',
```

Comment:
This is a good suggestion, I'll implement this as I build my features :+1: Only reason why I'm not doing it right now is because the focus of this PR is more so to get the scaffolding work done. @sourcery-ai